js/gratipay/edit_team.js

```diff
@@ -0,0 +1,51 @@
+Gratipay.edit_team = {}
+
+Gratipay.edit_team.initForm = function() {
+    $form = $("#edit-team");
+    $buttons = $form.find("button");
+
+    $form.find(":input").each(function() {
+        // Save original values.
+        $(this).data('original', $(this).val());
+    });
+    $form.submit(Gratipay.edit_team.submitForm);
+}
+
+Gratipay.edit_team.submitForm = function(e) {
+    e.preventDefault();
+
+    var modified = false;
+
+    var data = new FormData();
+    $form.find(":input").each(function() {
+        if($(this).data("original") !== $(this).val()) {
+            modified = true;
+
+            var name = $(this).attr("name");
+            if (name === "image") {
+                data.append(name, $(this)[0].files[0]);
+            } else {
+                data.append(name, $(this).val());
```

Comment:
> our JS logic fails to recognize that the field has been edited

Why would it, if it's written to do exactly that? Can you give an example where that happens?

I think you are concerned about https://github.com/gratipay/gratipay.com/pull/3923#issuecomment-189099923 where the form is submitted before the js even loads, which *may* be possible. And to fix that, a simple `enctype = multipart/form-data` on the form is enough. (both on new team and edit team forms). But I still think it's better to send only necessary data if we can.